### DISCUSSION TITLE
Github Copilot bug in the Student Developer Pack

### DISCUSSION THREAD

**Discussion by lpestov** [community/community#147748] [GitHub Education] [answered]
### Hi everyone,
I recently updated my student benefit pack, and on December 18, I received the final confirmation that all the features were available. 

![image](https://github.com/user-attachments/assets/43a34503-ea3c-447a-811c-45821f3ed2ad)

But I noticed that my AI assistant from github is still in the free version, but it is very limited, and before these restrictions were not there.
![image](https://github.com/user-attachments/assets/cc482dab-1296-4558-9fb1-9cafae03db40)
I can assume that this is due to the release of GitHub Copilot for all users (before that it was paid and had no restrictions), but it seems that the developers forgot to add the Pro version to the student pack.

**Comment by IrviWade** [comment, marked as answer]
> ### Hi everyone,
> I recently updated my student benefit pack, and on December 18, I received the final confirmation that all the features were available.
> 
> ![image](https://private-user-images.githubusercontent.com/74723645/397999516-43a34503-ea3c-447a-811c-45821f3ed2ad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ4ODY5OTksIm5iZiI6MTczNDg4NjY5OSwicGF0aCI6Ii83NDcyMzY0NS8zOTc5OTk1MTYtNDNhMzQ1MDMtZWEzYy00NDdhLTgxMWMtNDU4MjFmM2VkMmFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIyVDE2NTgxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwNDA5Y2I1ZmYzYTM2ZGRjNWU5YzUwMDYxMjMyMGJlOTFkMjE0YzYxYmE5MzRkZDM4ODRiODJjM2FlMTc3YzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.N1aypTSgMDSeV84tCSwOwI6bKd0MiO1Lo0XphJAEynQ)
> 
> But I noticed that my AI assistant from github is still in the free version, but it is very limited, and before these restrictions were not there. ![image](https://private-user-images.githubusercontent.com/74723645/397999562-cc482dab-1296-4558-9fb1-9cafae03db40.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ4ODY5OTksIm5iZiI6MTczNDg4NjY5OSwicGF0aCI6Ii83NDcyMzY0NS8zOTc5OTk1NjItY2M0ODJkYWItMTI5Ni00NTU4LTlmYjEtOWNhZmFlMDNkYjQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIyVDE2NTgxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzMDQxZTBiOTRlNzk2N2M5NTk0ZDUwZDBiOTQwYTM0NmFhNjQ1N2NjODNlNjU2ZTdjNGIxYzVkNGJiMDljYjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.nn77t1cioYeBad5NryJCxiGaG_GC2T05cIj4D_F785Q) I can assume that this is due to the release of GitHub Copilot for all users (before that it was paid and had no restrictions), but it seems that the developers forgot to add the Pro version to the student pack.

Hey! 🤔 You can try through : https://github.com/github-copilot/free_signup ^^

**Reply by lpestov** [reply]
Hey!
Lol, it worked. It is very strange that the plan did not change automatically, but it was necessary to follow some link. (I didn't know where to find it)

![image](https://github.com/user-attachments/assets/338b0bf9-3794-4cd8-a346-06403017c146)

Thank you very much!

**Reply by ivanjpj** [reply]
Thank you!
